assets/templates/app/app.ts

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+import Homey from 'homey';
```

Comment:
is this `allowSyntheticDefaultImports`? We should decide whether we want a single `Homey` default export or named exports (also given that node has added ESM support)

Reply:
this is due to the `"esModuleInterop": true` flag.

I like the es6 syntax more than commonJS but I guess we should support both. What do you suggest?

Reply:
No I mean I that I was confused because this only works if we enabled the synthetic default imports compiler options somewhere but the config didn't have it, turns out `esModuleInterop` enables `allowSyntheticDefaultImports` automatically.

The point of the comment however is that we need to decide wether the SDK should behave as a default export or named exports. If we choose the latter this import should be `import * as Homey from 'homey'`. I think I prefer that the DSK exports `Homey` as a default export, but this means the other templates should not use `import { Device } from 'homey'`.